subsys/net/openthread/Kconfig.defconfig

```diff
@@ -24,7 +24,7 @@ config SYSTEM_WORKQUEUE_STACK_SIZE
 
 config MAIN_STACK_SIZE
 	int
-	default 2048
+	default 4096
```

Comment:
That's quite a big increase. Have you checked what value is actually in needed by the samples? You can use thread analyzer (CONFIG_THREAD_ANALYZER) for this purpose?

Reply:
I've hit the same issue upstream, indeed it's not needed to increase the stack size that much
> 0x20003590 main       (real size 2560): unused 408      usage 2152 / 2560 (84 %)

Reply:
Looks like the max utilization currently is 2104 so I increased the size to 2560.